azure-kit-starter/src/main/java/com/vaadin/azure/starter/sessiontracker/serialization/TransientHandler.java

```diff
@@ -0,0 +1,10 @@
+package com.vaadin.azure.starter.sessiontracker.serialization;
+
+/**
+ * A combination of inspector and injector.
+ * 
+ * TODO: maybe just drop extended interfaces and merge them into this one
+ */
+public interface TransientHandler
+        extends TransientInspector, TransientInjector {
+}
```

Comment:
They were born separately, but it seems like there is no good reason to have different interfaces, since `injector` needs to know how `inspector` is encoding transient instance reference identifier

Reply:
I'll drop `TransientInspector` and `TransientInjector` and merge them into `TransientHandler`

Reply:
done